messages_zones.go

```diff
@@ -36,7 +36,7 @@ type ZoneOwner struct {
 type ZoneMeta struct {
 	Step                    int    `json:"step,omitempty"`
 	PageRuleQuota           string `json:"page_rule_quota,omitempty"`
-	CustomCertificateQuota  int    `json:"custom_certificate_quota,omitempty"`
+	CustomCertificateQuota  string `json:"custom_certificate_quota,omitempty"`
```

Comment:
When running against the actual API it returns a string for our zones listing in the form of:

``` json
"meta": {
  "step": 4,
  "wildcard_proxiable": false,
  "custom_certificate_quota": "1",
  "page_rule_quota": "50",
  "phishing_detected": false,
  "multiple_railguns_allowed": false
},
```